spec/features/admin_company_management_spec.rb

```diff
@@ -32,6 +32,7 @@ def login
       expect(page).to have_content(companies.first.email)
       expect(page).to have_content(companies.first.city)
       expect(page).to have_content(companies.first.state)
+      expect(page).to have_content(companies.first.status.capitalize)
```

Comment:
Or maybe should we just store the status as capitalized words since we use them as capitalized anyways? What would be more expected and readable? I really think it would be fine either way but this may solve some steps in not having to downcase and then re-upcase